assets/less/components/donation-elements.less

```diff
@@ -1,30 +1,49 @@
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
 .modal {
-    width:50%;
-    margin-left:auto;
-    margin-right:auto;
-    padding: 5px;
-    border: thin #000 solid;
-    background-color:#fff;
-    z-index:3; /* places the modal dialog on top of everything else */
-    position:fixed;
-    top:25%;
-    left:25%;
-    display:none;
-}
+    top: calc(50% - 250px);
```

Comment:
I would use
```
top: 50%;
transform: translateY(-50%);
```
So it's vertically centered in the page and if the browser height is too narrow, the important content is still visible at the center.